docs/resources/files.md

```diff
@@ -0,0 +1,54 @@
+---
+subcategory: "Storage"
+---
+# databricks_files Resource
+
+This is a resource that lets you upload and download files upto 2GB in octet-stream.
+
+## Example Usage
+
+In order to manage a file on Databricks File System with Terraform, you must specify the `source` attribute containing the full path to the file on the local filesystem.
+
+```hcl
+resource "databricks_files" "this" {
+  source = "${path.module}/main.tf"
+  path   = "/tmp/main.tf"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `source` - The full absolute path to the file. Conflicts with `content_base64`.
+* `path` - (Required) The path of the file in which you wish to save.
+
+## Attribute Reference
+
+In addition to all arguments above, the following attributes are exported:
+
+* `id` - Same as `path`.
+* `file_size` - The file size of the file that is being tracked by this resource in bytes.
+* `modification_time` - The last time stamp when the file was modified
+
+
+## Import
+
+The resource dbfs file can be imported using the path of the file:
+
+```bash
+$ terraform import databricks_files.this <path>
+```
+
+## Related Resources
+
+The following resources are often used in the same context:
+
+* [databricks_workspace_file](./workspace_file.md)
```

Comment:
confusing, 

databricks workspace file
dbfs file
databricks files

confusing -- what should customers use? and what is the difference? This needs to be documented clearly